problemsets-mat223/linearalbegra.tex

```diff
@@ -2208,8 +2236,28 @@ \section*{Projections}
 	\question
 	Let $\vec a,\vec b\in \R^3$ be unknown vectors.
 	\begin{parts}
-		\item List two conditions that $\Comp_{\vec b} \vec a$ must satisfy.
+		\item List two conditions that $\Comp_{\vec b}\vec a$ must satisfy.
+			\begin{solution}
+				$\Comp_{\vec b} \vec a$ must be a scalar multiple of $\vec b$.
+
+				$\vec a - \Comp_{\vec b} \vec a$ must be orthogonal to $\vec b$,
+				or in other words $(\vec a-\Comp_{\vec b}\vec a)\cdot \vec b=0$. 
+			\end{solution}
 		\item Find a formula for $\Comp_{\vec b}\vec a$.
+			\begin{solution}
+				$\Comp_{\vec b}\vec a = \dfrac{\vec a\cdot\vec b}{\norm{\vec b}^2} \vec b$.
```

Comment:
I'd like `b dot b` on the bottom

Reply:
Sure. Do you want me to make that change everywhere? I think I used the square norm every time I used that formula for \Comp.